spyder/plugins/debugger/panels/debuggerpanel.py

```diff
@@ -8,7 +8,7 @@
 """
 # Third party imports
 from qtpy.QtCore import QSize, Qt, QRect, Slot
-from qtpy.QtGui import QPainter, QFontMetrics
+from qtpy.QtGui import QPainter, QFontMetrics, QColor
```

Comment:
```suggestion
from qtpy.QtGui import QPainter, QFontMetrics
```

This import is unused now.